pkg/cri/server/sandbox_run.go

```diff
@@ -379,7 +379,7 @@ func (c *criService) setupPodNetwork(ctx context.Context, sandbox *sandboxstore.
 	if err != nil {
 		return errors.Wrap(err, "get cni namespace options")
 	}
-
+	log.G(ctx).Debugf("begin cni setup for pod id %q", id)
```

Comment:
We should use `WithField` for identifiers. I know the cri package currently doesn't make good use of it now but we should enforce on new lines.

Reply:
makes good sense... made use of WithField for pod sandbox id debug out's where appropriate, at least in the touched file, Cheers!